controllers/nutanixmachine_controller.go

```diff
@@ -666,7 +666,7 @@ func (r *NutanixMachineReconciler) getOrCreateVM(rctx *nctx.MachineContext) (*nu
 		return nil, errorMsg
 	}
 	log.Info(fmt.Sprintf("Waiting for task %s to get completed for VM %s", lastTaskUUID, rctx.NutanixMachine.Name))
-	err = nutanixClient.WaitForTaskCompletion(ctx, nc, lastTaskUUID)
+	err = nutanixClient.WaitForTaskToSucceed(ctx, nc, lastTaskUUID)
```

Comment:
nit: can we inline this?
```go
if err := ...; err != nil {
    ...
}
```

Reply:
The rest of the function is not inlining it, since this was existing code can we make it consistent through a linter instead?

Reply:
We'll have to do that in other PRs, then. But, sure, as long as we do get around to it.